api/envoy/config/route/v3/route.proto

```diff
@@ -122,30 +121,16 @@ message RouteConfiguration {
   //
   google.protobuf.UInt32Value max_direct_response_body_size_bytes = 11;
 
-  // [#not-implemented-hide:]
   // A list of plugins and their configurations which may be used by a
-  // :ref:`envoy_v3_api_field_config.route.v3.RouteAction.cluster_specifier_plugin`
-  // within the route. All *extension.name* fields in this list must be unique.
+  // :ref:`cluster provider name <envoy_v3_api_field_config.route.v3.RouteAction.cluster_provider_name>`
```

Comment:
Please restore this comment to its original text.  The new text here doesn't match the actual fields.

Reply:
The comments update is necessary to fix the doc link.
But I copied the comments in my old commit directly which results in some inconsistent content. I will fix it. Thanks.